packages/ember-auto-import/ts/splitter.ts

```diff
@@ -17,7 +17,7 @@ const debug = makeDebug('ember-auto-import:splitter');
 const resolver = ResolverFactory.createResolver({
   // upstream types seem to be broken here
   fileSystem: new CachedInputFileSystem(new NodeJsInputFileSystem(), 4000) as unknown as AbstractInputFileSystem,
-  extensions: ['.js', '.json'],
+  extensions: ['.js', '.ts', '.json'],
```

Comment:
The order in this array takes precedence over the order defined in [`webpack.resolve.extensions`](https://github.com/ef4/ember-auto-import/blob/89b72ea947e3fe35f1af91ba9df4d1a0bf83c797/packages/sample-typescript/ember-cli-build.js#L9-L11).